tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
     "compilerOptions": {
         "noEmit": true,
```

Comment:
tsconfig for Parcle-bygg. La Parcel fikse bygging, TypeScript skal kun typesjekke